### ISSUE TITLE
SIGSEGV when doing OpenIMAJGrabber.setTimeout(..) if grabbing session is not started

### ISSUE THREAD

**Issue by sarxos** [openimaj/openimaj#79]
Hello @openimaj Team!

Today I was doing some tests and suddenly caught this issue. It's very low priority since ```OpenIMAJGrabber``` is protected in the package so it cannot be used directly. Just thought to let you know so you are aware.

```plain
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fae277de03b, pid=12463, tid=140387422983936
#
# JRE version: Java(TM) SE Runtime Environment (8.0_31-b13) (build 1.8.0_31-b13)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.31-b07 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [OpenIMAJGrabber.so+0x203b]  OpenIMAJGrabber::setTimeout(int)+0x15
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/sarxos/workspace/webcam-capture/webcam-capture/hs_err_pid12463.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

Test code:

```java
package org.openimaj.video.capture;

public class Test {

	public static void main(String[] args) {
		OpenIMAJGrabber grabber = new OpenIMAJGrabber();
		grabber.setTimeout(5000);
	}
}
```

I think the solution would be simply to check if ```VG``` is not ```null``` in ([link](https://github.com/openimaj/openimaj/blob/master/hardware/core-video-capture/src-native/linux/OpenIMAJGrabber.cpp#L156)):

```cpp
void OpenIMAJGrabber::setTimeout(int timeout) {
	VG->timeout = timeout;
}
```
